pack-hdf-converters.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
why does this not have any of the echo lines that the windows version does?

Reply:
Added them in just now (originally the windows' echos were just for debugging, but we kept them in because we thought it would be more easily maintainable. never got around to adding them to the .sh because it worked the first time around)